plugins/restful/RestfulDataProviderDbQuery.php

```diff
@@ -349,25 +349,43 @@ public function update($id, $full_replace = FALSE) {
     // Build the update array.
     $request = $this->getRequest();
     static::cleanRequest($request);
+    $save = FALSE;
+    $original_request = $request;
+
     $public_fields = $this->getPublicFields();
     $fields = array();
-    foreach ($public_fields as $public_property => $info) {
+    foreach ($public_fields as $public_field_name => $info) {
+      if (!empty($info['create_or_update_passthrough'])) {
+        // Allow passing the value in the request.
+        if (!empty($info['create_or_update_passthrough_required']) && !isset($request[$public_field_name])) {
```

Comment:
> We are deferring validation (for entities) to Entity Validation. So I'm not sure why we need to validate input data for the passthrough fields.

You mean you don't like the ``required`` property? In that case where do you think we should we move it?